### PR TITLE
Schema for Santa Clara, CA

### PR DESCRIPTION
**Pull request by RickHawesUSDS**
This PR adds a schema for Santa Clara, CA.

## Changes
- SCC schema
- SCC org files

## Checklist
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [ ] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from http://localhost:7071/api/download
- [ ] Updated the release notes? 
- [ ] Added tests?
- [ ] Did you check for sensitive data, and remove any? 
- [ ] Does logging contain sensitive data?  
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Security
*List potential security threats and mitigations in the PR*

## Fixes
*List GitHub issues this PR fixes*
- #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

File: prime-router/metadata/schemas/covid-19.schema [hunk 1]
```diff
@@ -152,6 +152,7 @@ elements:
     type: ID
     hl7Field: ORC-3-1
     hl7OutputFields: [ ORC-3-1, SPM-2-2, OBR-3-1 ]
+    documentation: Assension number
```
Comment (MauriceReeves-usds) [inline prime-router/metadata/schemas/covid-19.schema:155]: Should be spelled `accession number`

File: prime-router/metadata/schemas/covid-19.schema [hunk 2]
```diff
@@ -841,6 +842,7 @@ elements:
     hl7Field: SPM-10
     referenceUrl: https://hl7-definition.caristix.com/v2/HL7v2.5.1/Fields/SPM.10
 
+  # should this also have a lookup into LIVD?
   - name: specimen_description
```
Comment (MauriceReeves-usds) [inline prime-router/metadata/schemas/covid-19.schema:846]: We could add it to LIVD, but I don't think that the description is dependent on the type of device used like other fields are.

File: prime-router/metadata/schemas/CA/ca-scc-covid-19.schema [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+      - name: "Race"
+        format: "$code"
+
+  - name: "patient_ethnicity"
+    csvFields:
+      - name: "Ethnicity"
+        format: "$code"
+
+  - name: "local_Language"
+    default: ""
```
Comment (MauriceReeves-usds) [inline prime-router/metadata/schemas/CA/ca-scc-covid-19.schema:70]: for the purposes of faking data, all elements should have a type defined, or you will encounter the error on line 170 in `FakeReport` about invalid element type.

File: prime-router/metadata/schemas/CA/ca-scc-covid-19.schema [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    csvFields:
+      - name: "Patient Last Name"
+
+  - name: "patient_dob"
+    csvFields:
+      - name: "Patient Date of Birth"
+
+  - name: "patient_gender"
+    csvFields:
+      - name: "Patient Sex"
```
Comment (MauriceReeves-usds) [inline prime-router/metadata/schemas/CA/ca-scc-covid-19.schema:57]: Are we passing through the text value or the code for this?

File: prime-router/metadata/schemas/CA/ca-scc-covid-19.schema [hunk 5]
```diff
@@ -0,0 +1,205 @@
+name: "ca-scc-covid-19"
+topic: "covid-19"
+description: "CSV download schema for Santa Clara County Public Health Department"
+trackingElement: filler_order_id
+basedOn: "covid-19"
```
Comment (MauriceReeves-usds) [inline prime-router/metadata/schemas/CA/ca-scc-covid-19.schema:5]: If we have a link for their schema it's good to put the `referenceUrl` at the top.

File: prime-router/metadata/schemas/CA/ca-scc-covid-19.schema [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+      - name: "Patient County"
+
+  - name: "patient_phone_number"
+    csvFields:
+      - name: "Patient Phone Number"
+
+  - name: "local_OK_to_Contact_Patient"
+    default: ""
+    csvFields:
+      - name: "OK to Contact Patient"
```
Comment (MauriceReeves-usds) [inline prime-router/metadata/schemas/CA/ca-scc-covid-19.schema:101]: Needs a type if we're going to run it through the faker

**Review by MauriceReeves-usds — CHANGES_REQUESTED** [review]
It would be good to add Santa Clara to `quick-test.sh` as part of this check in.

**Comment by RickHawesUSDS** [comment]
@MauriceReeves-usds Thanks for the suggestions. Improvements made.

File: prime-router/metadata/schemas/covid-19.schema (hunk 1 above)
Comment (RickHawesUSDS) [reply prime-router/metadata/schemas/covid-19.schema]: Fixed

**Review by MauriceReeves-usds — APPROVED** [review]

**Pull request merged**